src/legacy/ui/public/utils/collection.ts

```diff
@@ -36,7 +36,7 @@ export function move(
   objs: object[],
   obj: object | number,
   below: number | boolean,
-  qualifier: (object: object, index: number) => any
+  qualifier: (object: object, index: number) => any = () => {}
```

Comment:
wrong type. Please wait until  #53819 will be merged. Or use the same type for `qualifier` argument